other/bootstrap_daemon/src/tox-bootstrapd.c

```diff
@@ -175,42 +175,47 @@ static void daemonize(LOG_BACKEND log_backend, char *pid_file_path)
     }
 }
 
-void print_log(void *context, LOGGER_LEVEL level, const char *file, int line,
-               const char *func, const char *message, void *userdata)
-{
-    if (MIN_LOGGER_LEVEL != LOG_TRACE && MIN_LOGGER_LEVEL != LOG_DEBUG) {
-        return;
-    }
+// Logs toxcore logger message using our logger facility
 
+void toxcore_logger_callback(void *context, LOGGER_LEVEL level, const char *file, int line,
+                             const char *func, const char *message, void *userdata)
+{
     char *strlevel;
+    LOG_LEVEL log_level;
```

Comment:
Can we use the already exist argument `level` here?

Reply:
Right, `LOGGER_LEVEL` is toxcore's log level from `/toxcore/logger.h`, but `LOG_LEVEL` is tox-bootstrap's log level from `other/bootstrap_daemon/src/log.h`.